.jules/palette.md

```
@@ -0,0 +1,3 @@
+## 2026-04-11 - TUI Help Footer UX Polish
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)